src/App.tsx

```diff
@@ -8,7 +8,25 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
 export const App: React.FC = () => {
+  const [loading, setLoading] = useState(true);
+  const [currentTodo, setCurrentTodo] = useState<Todo | null>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [allTodos, setAllTodos] = useState<Todo[]>();
```

Comment:
Consider initializing `allTodos` with an empty array `[]` instead of `undefined`. This will prevent potential issues when accessing `allTodos` before it is set.